uber/templates/jobs/index.html

```diff
@@ -68,20 +68,20 @@
         </div>
 
 </div>
-{% comment %}
+{# comment
```

Comment:
Agreed -- Git history means we don't really need to comment stuff out like this.